packages/console/src/components/Tooltip/index.module.scss

```diff
@@ -1,25 +1,28 @@
 @use '@/scss/underscore' as _;
 
-.container {
+.tooltip {
   position: absolute;
-  transform: translate(-50%, -100%);
   border-radius: 8px;
-  background: #191c1d;
-  color: #e0e3e3;
-  box-shadow: var(--light-shadow-s1);
+  background: var(--color-on-background);
+  color: var(--color-background);
```

Comment:
These color sets will not work well in dark theme. Please contact Guamian to update

Reply:
Leave it to a later iteration.